R/geom_phylopic.R

```diff
@@ -68,6 +68,11 @@ geom_phylopic <- function(mapping = NULL, data = NULL,
   if (!is.logical(remove_background)) {
     stop("`remove_background` should be a logical value.")
   }
+  # We need to do this before passing to the layer
+  dots <- list(...)
+  if ("img" %in% names(dots) && (is(dots$img, 'Picture') || is.array(dots$img))) {
```

Comment:
Is `&& (is(dots$img, 'Picture') || is.array(dots$img))` strictly necessary here? Shouldn't this be handled as an error elsewhere? If not, shouldn't we have error handling here? `img` should conform to one of these?

Reply:
You're right, this checking is already handled in GeomPhylopic, so I've changed this to just check if it is a list already.